yarn.lock

```diff
@@ -2537,6 +2537,10 @@ hoek@2.x.x:
   version "2.16.3"
   resolved "https://registry.yarnpkg.com/hoek/-/hoek-2.16.3.tgz#20bb7403d3cea398e91dc4710a8ff1b8274a25ed"
 
+hoist-non-react-statics@^1.2.0:
```

Comment:
Interesting... okay, good to know. 
It sounds like it may provide some general performance benefits too.